src/common/low_precision_transformations/tests/multiply_transformation.cpp

```diff
@@ -20,14 +20,49 @@
 #include "simple_low_precision_transformer.hpp"
 #include "lpt_ngraph_functions/multiply_function.hpp"
 
+#include "ngraph/pass/serialize.hpp"
+
 namespace {
 using namespace testing;
 using namespace ov;
 using namespace ov::pass;
 using namespace ngraph::builder::subgraph;
 
+class MultiplyBranch {
+public:
+    ngraph::builder::subgraph::Constant constant;
+    ngraph::element::Type input_precision;
+    ngraph::builder::subgraph::DequantizationOperations dequantization;
+};
+
+inline std::ostream& operator<<(std::ostream& out, const MultiplyBranch& branch) {
+    if (branch.constant.empty()) {
+        out << "_input=" << branch.input_precision;
+    } else {
+        out << "_constant=" << branch.constant;
+    }
+    return out << "_" << branch.dequantization;
+}
+
+class MultiplyValues {
+public:
+    MultiplyBranch branch1;
+    MultiplyBranch branch2;
+    ngraph::builder::subgraph::DequantizationOperations after_dequantization;
+};
+
+inline std::ostream& operator<<(std::ostream& out, const MultiplyValues& values) {
+    return out << "_branch1=" << values.branch1 << "_branch2=" << values.branch2 << "_after=" << values.after_dequantization;
+}
+
 class MultiplyTransformationTestValues {
 public:
+    // use this value in test case declaration to set precision as input precision
+    static const ov::element::Type input_precision;
+
+    // use this value in test case declaration to set precision as model precision
+    static const ov::element::Type model_precision;
```

Comment:
Looks like I missed something but I don't understand why `input_precision` and `model_precision` should be `static const`. Could you please elaborate on that?

Reply:
`input_precision` & `model_precision` are markers which are used in test case instantiations. Some used in test case instantization structures require precisions which will be defined from test parameterization. In test implementation these values are updated from tests parameters. This approach allows to parametirize test cases in simplier way.

Reply:
The purpose of these values is clear, thanks